raiden-ts/src/transport/epics.ts

```diff
@@ -94,45 +94,52 @@ const userRe = /^@(0x[0-9a-f]{40})[.:]/i;
 export const initMatrixEpic = (
   {  }: Observable<RaidenAction>,
   state$: Observable<RaidenState>,
-  { address, network, signer, matrix$ }: RaidenEpicDeps,
+  { address, network, signer, matrix$, config$ }: RaidenEpicDeps,
 ): Observable<ActionType<typeof matrixSetup>> =>
   state$.pipe(
     first(),
     mergeMap(state => {
```

Comment:
You should use `withLatestFrom(config$),` before this:
```suggestion
    withLatestFrom(config$),
    mergeMap(([state, config]) => {
```